### ISSUE TITLE
[ error] yourdomain/register 

### ISSUE THREAD

**Issue by gm8300130** [adevait/hrm#32]
Hi, 

It's sql error, 

I'm not sure if it's necessary, 

because `views/auth/register.blade`  without `how_did_they_hear`

so I set it to null

now it's work

```
2017_07_05_121947_update_table_users_add_column_how_did_they_hear.php


public function up()
    {
        Schema::table('users', function (Blueprint $table) {
            $table->string('how_did_they_hear')->after('notes')->nullable();
        });
    }
```

Thanks

**Comment by sandeepwelogical** [comment]
I have successfully setup the project on my localhost.
I not able to load this register page or login page

I am getting following error:



Sorry, the page you are looking for could not be found.

1/1
NotFoundHttpException in RouteCollection.php line 161:
in RouteCollection.php line 161
at RouteCollection->match(object(Request)) in Router.php line 766
at Router->findRoute(object(Request)) in Router.php line 621
at Router->dispatchToRoute(object(Request)) in Router.php line 607
at Router->dispatch(object(Request)) in Kernel.php line 268
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 104
at Pipeline->then(object(Closure)) in Kernel.php line 150
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 117
at Kernel->handle(object(Request)) in index.php line 54

**Comment by TsTrv** [comment]
@sandeepwelogical the problem seems to be with your local setup. Did you create a virtual directory?

**Comment by TsTrv** [comment]
I am closing this one. It is fixed under #27 and merged into `master`. Thanks, @jangelovski!
